test/test-utils/para-node.ts

```diff
@@ -156,7 +154,6 @@ export async function startParachainNodes(options: ParachainOptions): Promise<{
 				bin: BINARY_PATH,
 				id: (1000 * (i + 1)).toString(),
 				balance: "1000000000000000000000",
-				// chain: options.chain, this is not supported by cumulus
```

Comment:
@riusricardo this is the comment I was referring to. I'm going to put it back though, for collators that do need it